maths/sigmoid.py

```diff
@@ -0,0 +1,38 @@
+"""
+This script demonstrates the implementation of the Sigmoid function.
+
+The function takes a vector of K real numbers as input and then 1 / (1 + exp(-x)).
+After through Sigmoid, the element of the vector mostly 0 between 1. or 1 between -1.
+
+Script inspired from its corresponding Wikipedia article
+https://en.wikipedia.org/wiki/Sigmoid_function
+"""
+
+import numpy as np
+
+
+def sigmoid(vector: float):
```

Comment:
```suggestion
def sigmoid(vector: np.array):
```

Reply:
@xcodz-dot Thank you for your advice. I revised the code and uploaded it.

Reply:
😄 thanks